cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -0,0 +1,243 @@
+name "AWS Delete Unused Elastic Load Balancers (CLB)"
```

Comment:
I feel like we should just refer to "Classic Load Balancer" everywhere as opposed to ELB (CLB). Or at the very least, spell out the abbreviations at least once in each file to ensure there is a clear understanding of what the abbreviations are referring to. Not sure though. Interested if @cdwilhelm @ryanoleary @rshade @gonzalez have any additional thoughts.

Reply:
Pls let us know your thoughts on the name to be considered ?

Reply:
The policy template and relevant docs should be updated to reference "Classic Load Balancers" as opposed to ELB (CLB).

Reply:
As per the above comments, Updated the policy template and relevant docs. Thanks.